src/main/java/org/apache/commons/imaging/formats/tiff/write/TiffOutputField.java

```diff
@@ -66,6 +66,11 @@ public boolean bytesEqual(final byte[] data) {
         return Arrays.equals(bytes, data);
     }
 
+    /* Only used in tests. */
+    protected boolean bytesEqual(final TiffOutputField other) {
+        return Arrays.equals(bytes, other.bytes);
+    }
+
     protected AbstractTiffOutputItem getSeperateValue() {
```

Comment:
Oh, a typo. Will fix in another PR.

Reply:
I saw that too and decided it's out of scope. ;)